src/components/PoolingWidget/index.tsx

```diff
@@ -166,10 +173,11 @@ const PoolingInterface: React.FC = () => {
             setTxHash(txHash)
 
             batchedUpdates(() => {
+              let orderId = lastOrderIdRef.current + 1
```

Comment:
I see why you do this, but would this work the second time I send an order?
Are the pending orders  also consider in the `useLastOrderIdRef(orders)`? If not, Is it possible that together with the change of not showing trades with the same id would make it to not display the second pending orders.

Reply:
You are right
I address it in 0270223

Reply:
I think we would still have the issue of the refresh making the new pending order to dissapear if we get confirmation on an old pending order that was lost because of the refresh https://github.com/gnosis/dex-react/pull/723#issuecomment-600049389